### ISSUE TITLE
interproscan 5.32-71.0 get stuck in "81% completed"

### ISSUE THREAD

**Issue by wingwingWY** [ebi-pf-team/interproscan#85]
Hi,
I ran interproscan for 2000 proteins, and  it got stuck in "81% completed".  The program is still running. I use the default interproscan.properties file.  How to solve this problem?
```
24/01/2019 12:18:44:840 Welcome to InterProScan-5.32-71.0
24/01/2019 12:19:08:934 Running InterProScan v5 in STANDALONE mode... on Linux
24/01/2019 12:19:22:277 Loading file /home/ww/proteins_0.fasta
24/01/2019 12:19:22:439 Running the following analyses:
[CDD-3.16,Coils-2.2.1,Gene3D-4.2.0,Hamap-2018_03,MobiDBLite-2.0,Pfam-32.0,PIRSF-3.02,PRINTS-42.0,ProDom-2006.1,ProSitePatterns-2018_02,ProSiteProfiles-2018_02,SFLD-4,SMART-7.1,SUPERFAMILY-1.75,TIGRFAM-15.0]
Pre-calculated match lookup service DISABLED.  Please wait for match calculations to complete...
24/01/2019 15:07:05:986 25% completed
24/01/2019 16:07:05:813 26% completed
24/01/2019 17:07:05:820 74% completed
24/01/2019 18:07:05:889 81% completed
24/01/2019 19:07:05:976 81% completed
24/01/2019 20:07:06:072 81% completed
24/01/2019 21:07:06:138 81% completed
24/01/2019 22:07:06:145 81% completed
24/01/2019 23:07:06:258 81% completed
25/01/2019 00:07:06:355 81% completed
25/01/2019 01:07:06:380 81% completed
25/01/2019 02:07:06:429 81% completed
25/01/2019 03:07:06:486 81% completed
25/01/2019 04:07:06:597 81% completed
25/01/2019 05:07:06:688 81% completed
25/01/2019 06:07:06:801 81% completed
25/01/2019 07:07:06:860 81% completed
25/01/2019 08:07:06:960 81% completed
25/01/2019 09:07:07:075 81% completed
25/01/2019 10:07:07:192 81% completed
```

**Comment by gsn7** [comment]
this is strange.  it might have to do with the machine you are running on. kill it and rerun the interproscan.

**Comment by gsn7** [comment]
this should be fixed in release 5.37-76.0
